General | Move Jessie systems to "jessie-support" branch

MichaIng opened the pull request:
**Status**: Review
- [x] DietPi-Patch | Apply the same there to migrate as well older DietPi systems without pre-patch support.
- [x] Changelog

**Reference**: https://github.com/MichaIng/DietPi/issues/2332

**Commit list/description**:
- Apply the branch migration via pre-patches and regular patch.
- Older systems do not yet run pre-patches, so we need to apply with `patch_file` as well to catch all.
- Newer systems will be migrated via pre-patches which is preferred to prevent download and load of possibly incompatible code with `patch_file`.

MichaIng committed:
v6.23

+ DietPi-Pre-patch | Move Jessie systems to "jessie-support" branch

MichaIng committed:
v6.23

+ DietPi-Patch | Move Jessie systems to "jessie-support" branch

MichaIng committed:
v6.23

+ CHANGELOG | Jessie support fading info

Fourdee commented on CHANGELOG.txt:
```diff
@@ -2,6 +2,12 @@
+- As a result we move Jessie systems to a dedicated "jessie-support" branch. Next DietPi v6.24 will still be merged into this branch as well, but for following updates it depends on how much additional related issues we face.
+- We highly recommend to migrate to Stretch as fast as possible.
+- For Ordoid C1 there is no Stretch image available since its kernel and firmware are not maintained to match minimum requirements. There are reports about successful "apt-get dist-upgrade" migrations and an ARMbian Stretch image is available that can be used as a basis for DietPi-PREP. But these systems face major limitations, especially no GPU (Xserver, Kodi, desktops) support is available. Rea ...
+
```
Perfect! Great write up on this :+1:

Fourdee commented:
@MichaIng 

Looks good. Nice write up :+1: 

I'll have a look for my C1 (again) tomorrow, will run a quick patch test on dev.

MichaIng committed:
v6.23

+ DietPi-Patch | Coding

MichaIng committed:
v6.23

+ DietPi-Pre-patch | Coding and add print info about "jessie-support" migration
